### ISSUE TITLE
Printer driver name repeated in list several times

### ISSUE THREAD

**Issue by andrewdbate** [elementary/switchboard-plug-printers#156]
### What Happened?

If you try to install a printer manually by selecting a manufacturer and then a driver, often there will be many duplicate entries in this list. See screenshot below: 
![repeated_driver_name](https://user-images.githubusercontent.com/2998569/154994730-fa609e33-4beb-429b-b4a6-574cebaea2b9.png)

I am guessing that although these names are duplicated, it might not be the same that all these entries refer to the same driver.

However, there is no way to tell them apart. Hovering over the driver name in the list or right-clicking it reveals no more information. So I am really unsure which to choose. I have to just pick one and hope it works, which is not user friendly.

### Steps to Reproduce

1. Open Settings and choose Printers.
2. Click + in the bottom left corner.
3. Choose a printer in the device list (it can be either local or network, it doesn't matter).
4. Choose a manufacturer in the left-hand column (e.g. Kyocera, but lots of manufacturers have repeated entries)
5. See lots of duplicates entries in the right-hand list.

### Expected Behavior

Each item in the right-hand list should either be unique, or there should be some way to find out more information about the driver so you can make an informed choice about which to install.

### OS Version

6.x (Odin)

### Software Version

Latest release (I have run all updates)

**Comment by aral** [comment]
Ran into this too. Also, given the hundreds of entries in the HP category, I literally cannot even find my printer again (did once, selected it, and it wansn‘t set up correctly while connected to its WiFi directly. Also, that WiFi connection is stable either I think because elementary OS cannot find an internet connection on it and so intelligently jumps to a WiFi network with one, disconnecting from the printer. I might have more luck if I bring it in the room and connect it via USB I guess.)

**Comment by andrewdbate** [comment]
So I dig into this a bit more. I wanted to see if each item in the list really did correspond to a different driver.

In the screenshot above you have `Kyocera FS-C5100DN (KPDL)` listed multiple times.

So I added the printer using the first `Kyocera FS-C5100DN (KPDL)` entry and then printed a test page. Here is it (I scanned it to attach to this issue):
![test_page_1](https://user-images.githubusercontent.com/2998569/156577517-d61258fd-a27e-4c21-a9b2-1f4a96b6a70f.png)

Then I removed the printer and added it again but this time choosing the second `Kyocera FS-C5100DN (KPDL)` entry in the list and printed a test page:
![test_page_2](https://user-images.githubusercontent.com/2998569/156579294-14645af4-f950-4cd7-b213-0e90986cb99c.png)

And again with the third `Kyocera FS-C5100DN (KPDL)` entry in the list:
![test_page_3](https://user-images.githubusercontent.com/2998569/156577730-3aa8da2b-1b8b-4251-b6d6-6ab13d768101.png)

Note that the driver listed in each test page is different. The first used driver `KC100EG.PPD`, the second used driver `KC100EE.PPD`, the third used driver `KC100ES.PPD`.

This confirms that each `Kyocera FS-C5100DN (KPDL)` entry in this list is in fact a different driver.

I wanted to know where this drivers come from. I went to https://www.openprinting.org/ and then clicked Downloads then PPD and then Kyocera (the make of my printer). This took me to this page:
https://www.openprinting.org/download/PPD/Kyocera/

Here is a screenshot of that page so you know what I'm looking at:

![open-printing-screenshot](https://user-images.githubusercontent.com/2998569/156578495-28a6057e-81a5-4e6e-a1b7-22af01214f25.png)

You have the different languages `de`, `en`, `es`, `fr`, `it`, `pt`. Within each directory is a file named `Kyocera_FS-C5100DN.ppd`.
So within the `de` directory we have [Kyocera_FS-C5100DN.ppd](https://www.openprinting.org/download/PPD/Kyocera/de/Kyocera_FS-C5100DN.ppd). Within the `en` directory we have
[Kyocera_FS-C5100DN.ppd](https://www.openprinting.org/download/PPD/Kyocera/en/Kyocera_FS-C5100DN.ppd). Within the `es` directory we have [Kyocera_FS-C5100DN.ppd](https://www.openprinting.org/download/PPD/Kyocera/es/Kyocera_FS-C5100DN.ppd), and so on.

Downloading and opening each of these files with a text editor we can see that each has a line starting with `*PCFileName:`.

The file [Kyocera_FS-C5100DN.ppd](https://www.openprinting.org/download/PPD/Kyocera/de/Kyocera_FS-C5100DN.ppd) within the `de` directory has the line:
```
*PCFileName: "KC5100EG.PPD"
```
The file [Kyocera_FS-C5100DN.ppd](https://www.openprinting.org/download/PPD/Kyocera/en/Kyocera_FS-C5100DN.ppd) within the `en` directory has the line:
```
*PCFileName: "KC5100EE.PPD"
```
The file [Kyocera_FS-C5100DN.ppd](https://www.openprinting.org/download/PPD/Kyocera/es/Kyocera_FS-C5100DN.ppd) within the `es` directory has the line:
```
*PCFileName: "KC5100ES.PPD"
```

Each of these strings corresponds to the line starting with `Driver:` on the printer test page.

Diffing these files (using your favorite diff tool such as SmartSynchronize) shows that the drivers are identical except for translated strings.

Note that each of these files contain a line to identify the language such as:
```
*LanguageVersion: German
```

So in summary, each entry in the above list is a different driver, and in the case of drivers originating from openprinting.org, these are actually the same drivers but for different languages.

The user interface needs to expose which driver you will be installing, i.e., `KC5100EG.PPD` vs `KC5100EE.PPD` vs `KC5100ES.PPD` vs etc. If nothing else, maybe this could be added to the tool-tip shown when you hover over the list item?

It would be even better if the language of the driver was also shown in the user interface. Then it would be immediately obvious to the user which driver to install (and thus require less technical knowledge from the user). So for the Kyocera printer shown above, you would know to install the KPDL English driver instead of the KPDL German driver, for example.

**Comment by andrewdbate** [comment]
Scrolling through the list, this seems to be an issue for the vast majority of printers. Here is another screenshot of the issue for another printer:
![9ecebe42038178e8](https://user-images.githubusercontent.com/2998569/156601889-a681ec60-188f-40e3-835e-40e96554a4eb.png)

**Comment by tintou** [comment]
Ah, thanks for the feedback, I'm fixing it with this: https://github.com/elementary/switchboard-plug-printers/pull/158
